scripts/setupvars/setupvars.sh

```diff
@@ -149,9 +152,10 @@ check_python_version () {
 } 
 
 python_version_to_check="$python_version"
-local_py_version="$( python3 -c 'import sys; print(".".join(map(str, sys.version_info[0:2])))' )"
-if [ "$local_py_version" = "$python_version_to_check" ]; then
-        python_version_to_check="$PYTHON_VERSION_MAJOR"
+if [ -z "$python_version_to_check"  ]; then
+    python_version_to_check="3"
+else
+    python_version_to_check="$python_version_major"
```

Comment:
is `python_version_major` really defined?
I suppose it's defined only after `check_python_version ` call which happens below.

Reply:
@ilya-lavrenov I am sure that the source code looked so strange and as a w/a. I decided to change it after review. Please check.
Enjoy :)